drivers/performance_measurement/app/models/performance_measurement/report.rb

```diff
@@ -40,6 +40,14 @@ class Report < SimpleReports::ReportInstance
       order(updated_at: :desc)
     end
 
+    def reporting_spm_id
+      @reporting_spm_id ||= clients.first.reporting_spm_id if clients.any?
```

Comment:
This is fine, but might be slightly cleaner as 
```suggestion
      @reporting_spm_id ||= clients&.first&.reporting_spm_id 
```